pom.xml

```diff
@@ -76,7 +88,17 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-release-plugin</artifactId>
-        <version>2.5</version>
+        <version>2.5.2</version>
+      </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
```

Comment:
this is not a dependency on findbugs, it is configuring the maven findbugs plugin so that when it is run it has by default the correct configuration and a stable version.